### PR TITLE
Chore merge from 846d660 to 4536f60

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/846d6606ec4eabe585dd0240e3d1b56256cfc8a6
  - Forgot something in the first commit. It goes with https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1855/commits/ac104da07ae79282a23719648bba3723c1cf2d82 to add `bootstrap_github_org` in our `config.yml`
- [x] https://github.com/twbs/bootstrap/commit/8a3540803015cf9e1a9406ad7ffbfb264ac72ec9
- [x] https://github.com/twbs/bootstrap/commit/9c1aa675978caae4db35671ae72a95db20744ed2
- [x] https://github.com/twbs/bootstrap/commit/1c05a887eacee27b785a6fc2403a43cae76dfae3
- [x] https://github.com/twbs/bootstrap/commit/1b83ab24fcd14a7ca413c732eebf04193a644c8f
- [x] https://github.com/twbs/bootstrap/commit/d17d2abb26cc5e73b7592e3386d21000793b845d
- [x] https://github.com/twbs/bootstrap/commit/702a3b6a61fab473de356fe3fe91edf2b8a3b6f7
- [x] https://github.com/twbs/bootstrap/commit/fd1ec97bd3fdc5b942f4fff60c6764e3e13db2a8
- [x] https://github.com/twbs/bootstrap/commit/ffbe6e9246767b997d9d8d8fbe454aa8f4315d58
- [x] https://github.com/twbs/bootstrap/commit/7399b130525374423e537f5f083d4e749607ed31
- [x] https://github.com/twbs/bootstrap/commit/8265927fc1dc5fd18c13d04d623f00a77590e4e0
- [x] https://github.com/twbs/bootstrap/commit/1ed86c905f37ae97fd2ee02cf9e25a30f0d6a47f
- [x] https://github.com/twbs/bootstrap/commit/eb10b7079c7dc8e11d4924396f22605d1a8db4a1
- [x] https://github.com/twbs/bootstrap/commit/980116e02fbc99990a5070118210c2495993041d
- [x] https://github.com/twbs/bootstrap/commit/1e33cfa272542a9facf03033d765a5a3d86d3a35
- [x] https://github.com/twbs/bootstrap/commit/39905711fab711ac6f99f8a987931bdee9376b66
- ~~https://github.com/twbs/bootstrap/commit/187153131255dc0b6f415036a85dd4f0ece10ed6~~: nothing to do here since we only display `.fw-bold` and `.fw-normal` in our docs
- [x] https://github.com/twbs/bootstrap/commit/927ccdf714fd7dff3daba927ada00f4f132ef954
- [x] https://github.com/twbs/bootstrap/commit/3fc305b2de2a86afa3660bf1aecb4a15ceb53ffd
- [x] https://github.com/twbs/bootstrap/commit/f40e76d1f77c020be337d4244c7cfb13a2b5bd9c
- [x] https://github.com/twbs/bootstrap/commit/a901027ab02f86a0dc6ca3472cb7b63c128a6314
- [x] https://github.com/twbs/bootstrap/commit/81582ad953afaffcef8b1abc2836b286924fbed1
- [x] https://github.com/twbs/bootstrap/commit/79e006b9346799b9f469f54161e7201e8e4657b9
- [x] https://github.com/twbs/bootstrap/commit/791c44cc6d556f05eb5acd0206c2aa6fe77e1a7d
- [x] https://github.com/twbs/bootstrap/commit/748702d00f623c9b3700a83b743429beeae85e29
- [x] https://github.com/twbs/bootstrap/commit/47580ff9ba0faa7fca7ed69b8219e9bcda17d922
- [x] https://github.com/twbs/bootstrap/commit/2d57b88902b93a67adc80e056a22edfd3155de0b
- ~~https://github.com/twbs/bootstrap/commit/f23e203bd0096d7a2da9eec2cd57aa6ba65bf252~~: nothing to do here since we don't want to deprecate our Dark offcanvas
- [x] https://github.com/twbs/bootstrap/commit/60714e32c76c23c407c291e9bdebcdf211d45222
- ~~https://github.com/twbs/bootstrap/commit/4536f60d95cc659f32e5509d6232826a801e1959~~: nothing to do; it was already done in Boosted

### CODE REVIEW

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
Few changes to handle before approving !

File: site/content/docs/5.3/examples/_index.md [hunk 1]
```diff
@@ -17,26 +17,41 @@ One of the missing examples has priority? Feel free to contact a member of the [
...
+      <p>
+        <strong>RTL is still experimental</strong> and will evolve with feedback. Spotted something or have an improvement to suggest?
+      </p>
+      <p><a href="{{ $.Site.Params.repo }}/issues/new/choose">Please open an issue.</a></p>
+    </div>
+  {{ end -}}
+  {{ range $i, $example := $entry.examples -}}
+    {{- $len := len $entry.examples -}}
+    {{ if (eq $i 0) }}<div class="row">{{ end }}
+      {{ if $entry.external }}
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.3/examples/_index.md:45]: Tried something in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1855/commits/11f1c6ee512d8cb4edcc3a92a3a0101d8c03b761, tell me what you think of it :)

File: site/content/docs/5.3/migration.md [hunk 2]
```diff
@@ -208,13 +208,15 @@ Learn more by reading the new [color modes documentation]({{< docsref "/customiz
...
     <ul>
+      <li><code>--bs-border-radius-2xl</code></li>
       <li><code>--bs-offcanvas-transition-duration</code></li>
     </ul>
   </details>
 
 - <details class="mb-2">
     <summary><span class="badge bg-danger">Breaking</span> Deprecated Sass variables:</summary>
     <ul>
+      <li><code>$border-radius-2xl</code></li>
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.3/migration.md:225]: Yep good catch. Done in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1855/commits/3aef7e4eaeaba4e4ef38750894f30a62a724e8bd

File: site/content/docs/5.3/customize/color.md [hunk 3]
```diff
@@ -250,31 +250,31 @@ Colors ending in `-rgb` provide the `red, green, blue` values for use in `rgb()`
...
         {{< markdown >}}`--bs-primary-border-subtle`{{< /markdown >}}
       </td>
     </tr>
     <tr>
       <td class="ps-0">
-        <div class="px-3 py-2" style="background-color: var(--bs-primary-text); color: var(--bs-body-bg);">Text</div>
+        <div class="py-3 fw-bold h5" style="color: var(--bs-primary-text)">Text</div>
       </td>
       <td>
         {{< markdown >}}`--bs-primary-text`{{< /markdown >}}
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.3/customize/color.md:280]: First part rather seems to be related to how Bootstrap handles `--bs-primary-*`.
For the second part, removed the section as discussed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1855/commits/0b894ebea2a25508f3aea691030d8b8aeb262354

**Review by louismaximepiton — APPROVED** [review]
👌

File: site/content/docs/5.3/customize/color.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (louismaximepiton) [reply site/content/docs/5.3/customize/color.md:280]: See https://github.com/twbs/bootstrap/pull/38136

**Pull request merged**